skimage/measure/fit.py

```diff
@@ -73,9 +107,21 @@ def estimate(self, data):
 
         Returns
         -------
-        success : bool
-            True, if model estimation succeeds.
+        model : Self or ``FailedEstimation``
+            An instance of the line model if the estimation succeeded.
+            Otherwise, a sentinel object will be returned and signal a failed
+            estimation. Testing the truth value of the failed estimation
+            sentinel will return ``False``. E.g.
```

Comment:
```suggestion
            An instance of the line model if the estimation succeeded.
            Otherwise, a ``FailedEstimation`` object is returned, which evaluates
            to False. To handle failed estimations, do:
```

I think jargon like "sentinel object" here may confuse users unnecessarily.

Reply:
Rewritten.